docker-compose.yaml

```diff
@@ -21,9 +22,18 @@ services:
     restart: always
     # This will check if your certificate is up for renewal every 12 hours as recommended by Let’s Encrypt
     entrypoint: "/bin/sh -c 'trap exit TERM; while :; do certbot renew; sleep 12h & wait $${!}; done;'"
- 
+
   registry:
     restart: always
     image: registry:2
     volumes:
       - /mnt/docker0storage/registry:/var/lib/registry
+      - /mnt/docker0storage/config:/var/lib/config # Where will we store the config file?
```

Comment:
I was intending for this to be the volume for our auth, but I think the `docker_auth` volumes below will suffice?

Reply:
` - /mnt/docker0storage/registry:/var/lib/registry` is for our  registry config right?